app/templates/_snippets.html

```diff
@@ -76,9 +76,9 @@
         <tbody>
         {% for resource in dataset.resources %}
           <tr>
-            <td><i class="icon-file-text-alt"></i><a href="#resource-{{resource.name}}">co2-mm-mlo</a></td>
+            <td><i class="icon-file-text-alt"></i><a href="#resource-{{resource.name}}">{{resource.name}}</a></td>
             <td></td>
-            <td>csv</td>
+            <td>{{ resource.format }}</td>
```

Comment:
it would have been nice to upper case this I think so it was consistent. Not absolutely sure and something of a detail -- but an example for you of consistently thinking about user experience.